templates/helpers.rb

```diff
@@ -314,6 +316,47 @@ def in_context name
     @convert_context.pop
     result
   end
+
+  STEM_EQNUMS_AMS = 'ams'
+  STEM_EQNUMS_NONE = 'none'
+  STEM_EQNUMS_VALID_VALUES = [
+    STEM_EQNUMS_NONE,
+    STEM_EQNUMS_AMS,
+    'all'
+  ]
+
+  # Generate the Mathjax markup to process STEM expressions
+  # @param cdn_base [String]
+  # @return [String]
+  def generate_stem(cdn_base)
+    if attr?(:stem)
+      eqnums_val = attr('eqnums', STEM_EQNUMS_NONE)
+      eqnums_val = STEM_EQNUMS_AMS if eqnums_val == ''
```

Comment:
Technically, I think we can remove this line, since empty `''` is not a valid value and thus will default to `STEM_EQNUMS_AMS` (in the code just below)

Reply:
You're right, fixed!